### PR TITLE
plugin/kubernetes: Add support for dual stack ClusterIP Services

### PR DESCRIPTION
**Pull request by chrisohaver**
### 1. Why is this pull request needed and what does it do?

This PR adds support for dual stack ClusterIP Services.

### CODE REVIEW

**Comment by chrisohaver** [comment]
fyi @aojea

File: go.mod [hunk 1]
```diff
@@ -32,12 +32,12 @@ require (
...
 	golang.org/x/crypto v0.0.0-20201002170205-7f63de1d35b0
-	golang.org/x/sys v0.0.0-20201015000850-e3ed0017c211
+	golang.org/x/sys v0.0.0-20201112073958-5cba982894dd
 	google.golang.org/api v0.29.0
 	google.golang.org/grpc v1.29.1
 	gopkg.in/DataDog/dd-trace-go.v1 v1.27.1
-	k8s.io/api v0.19.2
-	k8s.io/apimachinery v0.19.3
-	k8s.io/client-go v0.19.2
+	k8s.io/api v0.20.1-rc.1
```
Comment (aojea) [inline go.mod]: 0.20.0?? 
https://github.com/kubernetes/api/releases/tag/v0.20.0

Comment (chrisohaver) [reply go.mod]: yes - saw that - correcting now.

**Comment by chrisohaver** [comment]
This isn't merging well at all - most of the changes made are overwritten/lost, result of merge doesn't compile.  I _think_ it will be less headache to start over from  master.

**Comment by chrisohaver** [comment]
replaced by #4339

**Pull request closed without merging**